src/storage/keyvalue/JsonResourceStorage.ts

```diff
@@ -61,4 +62,8 @@ export class JsonResourceStorage implements KeyValueStorage<ResourceIdentifier,
       return false;
     }
   }
+
+  public entries(): never {
+    throw new NotImplementedHttpError();
```

Comment:
Why not needed?

Reply:
It's not that it's not needed, it's more that it is impossible to know which entries there are since it requires knowing all resources that are stored (and correspond to entries in case they are put in an existing other store). Will add comment.